addon/ng2/utilities/systemjs-helper.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../../typings/main.d.ts" />
+import fs = require('fs');
+
+export function loadSystemJson(systemPath: string): { [name: string]: any } {
+  const systemContents: any = fs.readFileSync(systemPath, 'utf8');
+  const jsonContent: any = systemContents.match(/^[^\{]*((?:.|\n)+)\);.*?$/m)[1];
```

Comment:
WRT the RegExp:

* The `?` at the end (before the `$`) seems redundant.

* `(?:.|\n)` will not match _any_ character (as seems to be the intention). For example, it will not match `\r`. So, users on Windows (where the newline sequence is `\r\n`) will run into errors.  
    If you want to match anything (including newline characters), then `[\s\S]` is the way to go (in JS at least).